std/tonumber.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package std
+
+import (
+	"fmt"
+	"strconv"
+
+	p "github.com/pulumi/pulumi-go-provider"
+	"github.com/pulumi/pulumi-go-provider/infer"
+)
+
+type Tonumber struct{}
+type TonumberArgs struct {
+	Input interface{} `pulumi:"input"`
+}
+
+type TonumberResult struct {
+	Result interface{} `pulumi:"result"`
```

Comment:
Shouldn't result here be `float64`?

Reply:
According to tf docs, `null` will return `null` , wasn't sure if we should pass `interface{}` vs `*float64`?